pywinauto/unittests/test_common_controls.py

```diff
@@ -441,6 +464,43 @@ class ListViewTestCases64(ListViewTestCases32):
         path = os.path.join(mfc_samples_folder, u"RowList.exe")
 
 
+class ListViewWinFormTestCases32(unittest.TestCase):
+

```

Comment:
You may try to remove this line to avoid Codacy warnings. I'd prefer keeping number of warnings non-growing. Fixing old warnings is optional, but sometimes we do it also.